shopinvader_quotation/services/quotation.py

```diff
@@ -63,9 +63,16 @@ def _validator_search(self):
     # All params are trusted as they have been checked before
 
     def _get_base_search_domain(self):
-        return self._default_domain_for_partner_records() + [
+        show_all = self.shopinvader_backend.quotation_expose_all
+        with_backend = not show_all
+        return self._default_domain_for_partner_records(with_backend=with_backend) + [
```

Comment:
this is actually wrong. I forgot to alter the domain to match SO that are bound to current backend or not bound at all (to not include orders bound to other backends

Reply:
fixed and added test coverage